lib/hanami/validations.rb

```diff
@@ -76,13 +45,13 @@ module ClassMethods
       # @see https://guides.hanamirb.org/validations/overview
       #
       # @example Basic Example
-      #   require 'hanami/validations'
+      #   require "hanami/validations"
       #
       #   class Signup
       #     include Hanami::Validations
       #
       #     validations do
-      #       required(:name).filled
+      #       required(:name).value(:string)
```

Comment:
If you want this change to still represent the same behaviour as before, then it should be `filled(:string)` (not sure how important this is to you but I just wanted to note it!)

Reply:
@timriley Thanks for letting me know 👍 https://github.com/hanami/validations/pull/180/commits/eef2c6da00000446c045fe94b951f27466a6911c